### PR TITLE
added people/ app skeleton

### PR DESCRIPTION
**Pull request by bernwang**

### CODE REVIEW

File: hknweb/people/models.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  picture             = models.CharField(max_length=255)
+  private             = models.BooleanField(default=True, null=False)
+  local_address       = models.CharField(max_length=255, default="")
+  perm_address        = models.CharField(max_length=255, default="")
+  grad_semester       = models.CharField(max_length=255, default="")
+  approved            = models.BooleanField()
+  failed_login_count  = models.IntegerField(default=0, null=False)
+  current_login_at    = models.DateTimeField()
+  mobile_carrier_id   = models.IntegerField()
+  sms_alerts          = models.BooleanField(default=False)
```
Comment (jameslzhu) [inline hknweb/people/models.py]: Probably don't need mobile carrier ID right now.

Comment (jameslzhu) [inline hknweb/people/models.py]: We can probably ditch the mobile carrier ID / sms alerts fields.

File: hknweb/people/models.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  last_name           = models.CharField(max_length=255, null=False)
+  username            = models.CharField(max_length=255, null=False)
+  email               = models.CharField(max_length=255, null=False)
+  crypted_password    = models.CharField(max_length=255, null=False)
+  password_salt       = models.CharField(max_length=255, null=False)
+  persistence_token   = models.CharField(max_length=255, null=False)
+  single_access_token = models.CharField(max_length=255, null=False)
+  perishable_token    = models.CharField(max_length=255, null=False)
+  phone_number        = models.CharField(max_length=255)
+  aim                 = models.CharField(max_length=255)
```
Comment (jameslzhu) [inline hknweb/people/models.py]: Uh, AIM?

File: hknweb/people/models.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  id                  = models.IntegerField(primary_key=True)
+  first_name          = models.CharField(max_length=255, null=False)
+  last_name           = models.CharField(max_length=255, null=False)
+  username            = models.CharField(max_length=255, null=False)
+  email               = models.CharField(max_length=255, null=False)
+  crypted_password    = models.CharField(max_length=255, null=False)
+  password_salt       = models.CharField(max_length=255, null=False)
+  persistence_token   = models.CharField(max_length=255, null=False)
+  single_access_token = models.CharField(max_length=255, null=False)
+  perishable_token    = models.CharField(max_length=255, null=False)
```
Comment (jameslzhu) [inline hknweb/people/models.py]: Mentioned before, but we may choose a different signin authentication method (like Django's builtin one). Idk how but it's probably somewhere in the docs [here](https://docs.djangoproject.com/en/2.0/topics/auth/). Also, I think Michelle did a bit with that in pull request #3.

Comment (MingweiSamuel) [reply hknweb/people/models.py]: Extending the default user: https://docs.djangoproject.com/en/2.0/topics/auth/customizing/#extending-django-s-default-user

**Comment by ochan1** [comment]
Closing since this exists already in current codebase
https://github.com/compserv/hknweb/blob/7490fbb166208ba14463445ff0149292ca3c37ba/hknweb/models.py#L21

**Pull request closed without merging**